docs/fides/docs/development/release_checklist.md

```diff
@@ -31,7 +31,6 @@ Next, run the following checks against the environment you've spun up using `fid
 ### API
 
 - [ ] Verify that the generated API docs are correct
-- [ ] Verify that the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated
```

Comment:
This is an important step — to complete this I'll usually:
- look through each PR in the increment to be released
- verify whether that PR had API changes
- if it did, was there a corresponding Postman collection update?

Reply:
per discussion in slack, seems like we should instead move this to https://github.com/ethyca/fides/blob/main/.github/pull_request_template.md?